codec/src/test/java/io/netty/handler/codec/compression/Lz4FrameEncoderTest.java

```diff
@@ -115,6 +110,7 @@ protected ByteBuf decompress(ByteBuf compressed, int originalLength) throws Exce
         return Unpooled.wrappedBuffer(decompressed);
     }
 
+    /*
```

Comment:
What's this commented-out code about?

Reply:
We should remove it... this test is not valid anymore as we not buffer internally